src/sentry/loader/browsersdkversion.py

```diff
@@ -35,7 +35,7 @@ def load_registry(path):
 def get_highest_browser_sdk_version(versions):
     full_versions = [x for x in versions if _version_regexp.match(x)]
     return (
-        six.binary_type(max(map(parse_version, full_versions)))
```

Comment:
At some point a `six.binary_type` audit needs to be done. Currently sentry-flake8 B308 recommends binary_type if you try and use str. It's correct under py2 but not py3-compatible.

We should definitely also change B308 to say something like "You most likely want six.text_type if you want unicode. Otherwise use six.binary_type for a bytestring."